config/config.ini

```diff
@@ -0,0 +1,9 @@
+[roommgr]
```

Comment:
Credentials must not be stored in the VCS. Alternatively mark the file as an example, e.g. config.ini.example and add the necessary remarks in the docs.

Reply:
done